docker-compose.yml

```
@@ -18,15 +18,68 @@ services:
+      - MB_DB_DBNAME=metabase
+      - MB_DB_PORT=5432
+      - MB_DB_USER=postgres
+      - MB_DB_PASS=${METABASE_DB_PASSWORD}
+      - MB_DB_HOST=metabase-db
+    depends_on:
+      - metabase-db
+    volumes:
+      - metabase-data:/metabase-data
+    networks:
+      - mued-net
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)